pom.xml

```diff
@@ -1423,34 +1452,12 @@
             </goals>
             <configuration>
               <includeScope>test</includeScope>
-              <outputFile>${test_classpath_file}</outputFile>
+	      <outputProperty>test_classpath</outputProperty>
             </configuration>
           </execution>
         </executions>
       </plugin>
 
-      <!-- This plugin reads a file into maven property. And it lets us write groovy !! -->
```

Comment:
This might be unneeded, but how do you know? even I dont' know what this plugin is about.